docs/quickstart.rst

```diff
@@ -94,11 +94,17 @@ Next will now upload these files to your Studio project.
 
 3. Initialize the server-side
 ------------------------------
-The next step is to initialize the server side with the client code and the initial global model. 
-In the Studio UI, navigate to the project you created in step one and click on the "Sessions" tab. Click on the "New Session" button. Under the "Compute package" tab, select a name and upload the generated package file. Under the "Seed model" tab, upload the generated seed file:
+The next step is to initialize the server side with the client code and the initial global model. In the Studio UI,
+
+1. Navigate to the project you created in step 1 and click on the "Sessions" tab. 
```

Comment:
Will this format as an itemized list when built with sphinx?

Reply:
Yes. However, I changed it to the recommended syntax using '#.' for ordered lists.